internals/handlers/utils.go

```diff
@@ -24,14 +26,15 @@ import (
 )
 
 const (
-	ExpectedStateErr     = "Error to generate a random State for OIDC Authentification"
-	InvalidStateErr      = "OIDC authentication invalid state"
-	TokenExchangeErr     = "OIDC authentication Failed to exchange token"
-	NoIDTokenErr         = "OIDC authentication No ID token found"
-	IDTokenVerifyErr     = "OIDC authentication Failed to verify ID token"
-	TokenName            = "token"
-	AllowedCookiePath    = "/"
-	parseGlobalVariables = false
+	ExpectedStateErr                = "Error to generate a random State for OIDC Authentification"
+	InvalidStateErr                 = "OIDC authentication invalid state"
+	TokenExchangeErr                = "OIDC authentication Failed to exchange token"
+	NoIDTokenErr                    = "OIDC authentication No ID token found"
+	IDTokenVerifyErr                = "OIDC authentication Failed to verify ID token"
+	TokenName                       = "token"
+	AllowedCookiePath               = "/"
+	parseGlobalVariables            = false
```

Comment:
what is this variable for ? (parseGlobalVariables)

Reply:
By default, when we retrieve situation or instance parameters, we do not parse them. Therefore, we use this variable everywhere the parsing of the retrieved parameters is not required.